modules/ROOT/pages/index.adoc

```diff
@@ -70,106 +70,202 @@ openssl version
 C:\Program Files\OpenSSL-Win64\bin
 ----
 
 .. Add this path to the environmental variables list, found in menu:System variables[Path].
 
 .. Now verify that Windows has registered the path of the binary.
-Once again, open the command prompt (`cmd`) -- it must be a fresh command prompt, as the earlier one will not have loaded the newly-added environmental variables -- and enter the following:
+Once again, open the command prompt (`cmd`) -- it must be a fresh command prompt, as the earlier one will not have loaded the newly added environmental variables -- and enter the following:
 +
+[source, shell]
 ----
 openssl version
 ----
 
-=== Install the Couchbase ODBC Driver
+== Setting up a Secure Connection
 
-. Download the `couchbase-odbc-1.1-win64.msi` file from https://packages.couchbase.com/releases/couchbase-odbc-driver/1.1/couchbase-odbc-1.1-win64.msi[] -- see the xref:release-notes.adoc[release notes page] for more details.
+Couchbase *strongly* recommends that you secure your ODBC connection using SSL.
+This ensures encryption of the communication between Power BI and the Couchbase server/Capella.
 
-. Double-click the file to open the *Couchbase ODBC Setup Wizard*.
 
-. Install the ODBC drivers from the downloaded file. 
-This installation provides both ANSI and Unicode drivers.
+[tabs]
+====
+Couchbase Capella::
++
+--
+You will need to obtain a connection string for your database,
+and generate an API key to plug into the ODBC connector.
 
+[[copy-connection-string]] To obtain the `connection string`.
 
-== Configure an ODBC Data Source
+. Sign in to your Capella instance as an Administrator.
+. Select the menu:Connect[] item from the top-level menu.
+. Make a copy of the `public connection string`.
++
+image::get-connection-string.png[copying connection string]
++
+. In the next section of the page, you will find the link to create a list of
+btn:[Allowed IP Addresses]. Click on the link to create an entry for the IP address of the machine upon which you're running Power BI.
++
+image::allowed-ip-addresses.png[creating-allowed-ip address]
++
+TIP: You can use the btn:[Add Current IP Address] button to fill in the address of the machine currently running the web console.
 
-You configure a data source name (DSN) for each CBAS scope or Capella columnar database you want to use in Power BI. 
+. [[setup-database-access]] Return to the *Connect* screen and click on the btn:[Database Access] link.
+Create a new database access entry, setting values for `bucket`, and `scopes`.
++
+image::create-database-access.png[Creating database access]
 
-=== Prerequisites
+. To set up a secure connection, you will need to retrieve the root certificate for your Capella instance.
++
+[lowerroman]
+.. Click on btn:[Security Certificate] in the left menu.
+.. Click on btn:[Download] to transfer a copy of the root certificate to your local machine.
++
+image::download-root-certificate.png[]
+--
 
-To configure an ODBC DSN, you supply:
+Couchbase Standalone::
```

Comment:
Should this be changed to Couchbase Server? We use this terminology in the DSN dialog box.

Reply:
The agreed wording is **self-managed Couchbase Server**
See, for example, https://docs.couchbase.com/scala-sdk/current/hello-world/start-using-sdk.html#before-you-start